numpy/lib/_arraypad_impl.pyi

```diff
@@ -44,10 +44,16 @@ _ModeKind: TypeAlias = L[
 # specific modes. Consider adding more overloads to express this in the future.
 
 # Expand `**kwargs` into explicit keyword-only arguments
+PadWidthType: TypeAlias = (
```

Comment:
this type alias is not public API (and this one doesn't exist even), so needs a a leading underscore

```suggestion
_PadWidth: TypeAlias = (
```

And naming-wise; a `*Type` suffix is conventionally used to indicate that the type alias is a `type`, for example `IntType: TypeAlias = type[int]`.